src/index.html

```diff
@@ -7,8 +7,44 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img 
+          src="./images/logo.png" 
+          alt="icon Moyo" 
+          class="logo_Moyo"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="nav_list">
+          <li class="nav_item"><a href="#" class="is-active">APPLE</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">SAMSUNG</a></li>
+          <li class="nav_item"><a href="#" class="nav_link">SMARTPHONES</a></li>
```

Comment:
You need to follow BEM naming, also you pay attention with additional classes. `is-active` should be used only for displaing underline
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item"><a href="#" class="nav__link is-active">APPLE</a></li>
          <li class="nav__item"><a href="#" class="nav__link">SAMSUNG</a></li>
          <li class="nav__item"><a href="#" class="nav__link">SMARTPHONES</a></li>
```